docs/modules/ROOT/pages/connect-to-cluster.adoc

```diff
@@ -54,15 +54,9 @@ xref:developer-guide.adoc[].
 
 - xref:hazelcast:clients:java.adoc#configuring-java-client[Java client]
 
-- link:https://github.com/hazelcast/hazelcast-nodejs-client/blob/master/DOCUMENTATION.md#3-configuration[Node.js client]
-
 - link:https://hazelcast.readthedocs.io/en/stable/configuration_overview.html[Python client]
 
-- link:https://pkg.go.dev/github.com/hazelcast/hazelcast-go-client#hdr-Configuration[Go client]
-
-- link:http://hazelcast.github.io/hazelcast-csharp-client/dev/doc/configuration.html[.NET client]
-
-- link:https://github.com/hazelcast/hazelcast-cpp-client/blob/v4.1.0/Reference_Manual.md#142-configuring-hazelcast-c-client[{cpp} client]
+- link:link:http://hazelcast.github.io/hazelcast-csharp-client/latest/doc/index.html[.NET client]
```

Comment:
I think we have a duplicated `link:` in the link here

Reply:
Fixed